deploy/operator/utils.sh

```diff
@@ -97,3 +97,23 @@ function wait_for_boolean_field() {
     echo "Value of field ${field} of object ${object} under namespace ${namespace} has never become true"
     return 1
 }
+
+function get_image_without_tag() {
```

Comment:
for those functions, just add a comment to document what they actually returns
e.g.:
```
# given "registry.ci.openshift.org/ocp/release:tag"
# return "release"
```

Or rather use templated-like examples:
```
# given "<registry>/<repository>/<project>:<tag>"
# return "<project>"
```
either way it will be good to have such simple doc